### PR TITLE
Fix typos.

### PR DESCRIPTION
**Pull request by danix800**
Fix typos in:
*  engine/userguide/storagedriver/aufs-driver.md
*  engine/userguide/storagedriver/btrfs-driver.md
*  engine/userguide/storagedriver/selectadriver.md

### CODE REVIEW

File: opensource/project/set-up-git.md [hunk 1]
```diff
@@ -55,39 +55,39 @@ target="_blank">docker/docker repository</a>.
 
 5. Open a terminal window on your local host and change to your home directory.
 
-    ```bash
```
Comment (danix800) [inline opensource/project/set-up-git.md]: The fence not formatted with an extra white space.

File: opensource/project/test-and-docs.md [hunk 2]
```diff
@@ -198,27 +198,30 @@ Git for Windows installation.  **Git Bash**, just as it sounds, allows you to
 run a Bash terminal on Windows.
 
 1.  If you don't have one open already, start a Git Bash terminal.
-    ```bash
```
Comment (danix800) [inline opensource/project/test-and-docs.md]: Referring to image, not a code block.

File: opensource/workflow/find-an-issue.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ experience, and priority. You can filter using one or more labels. The
 kind and experience labels are useful for new contributors.
 
 The following table describes the kind labels.
-<strong class="foobar">kind/bug</strong>
+<!-- strong class="foobar">kind/bug</strong -->
```
Comment (danix800) [inline opensource/workflow/find-an-issue.md]: Seems to be useless.

**Comment by johndmulhausen** [comment]
Thanks @danix800 ! I jumped in and fixed a few nits because it was faster than going back and forth. Appreciate the PR greatly. :)

**Pull request merged**